draft-iab-path-signals-collaboration.md

```diff
@@ -390,10 +390,11 @@ There is a tradeoff here between flexibility and ensuring the
 minimality of information in the future. The concern is that a fully
 generic data sharing approach between different layers and parties
 could potentially be misused, e.g., by making the availability of some
-information a requirement for passing through a network.
+information a requirement for passing through a network. This is
+undesirable. 
 
-This is undesirable, and our recommendation is to employ
-very targeted, minimal information carriage mechanisms.
+Therefore the used mechanisms needs to be well target to the kind of
+information carried as well as required level of trust and protection.
```

Comment:
```suggestion
Therefore, the signaling mechanisms need to be targeted to the kind of
information carried as well as required level of trust and protection.
```

I think the old text was perhaps better?

Reply:
I'm uncertain about the word minimal in the old text. I think minimal in the sense that his has to fit well is correct but it doesn't necessarily be to be absolutely minimal. But "fit well" is covered by the word "targeted" already; can we many just remove the word "minimal"?

Also I have a hard time that we more or less say extensibility is bad, while we have other IAB documents that underline the importance of extensibility in protocol design. I think my message would be more: use the right mechanism for the right purpose but don't try to build a all-fits-one approach.

Reply:
I think the original text was making the point that extensibility in this specific instance is bad, for reasons stated. This was also one of the points made in the audience of the APN BOF. 

Personally, I'd like to ccontinue to make the point. We can of course improve the text, but if we lose the point...  I think the draft is not as strong as it could be.